quickwit/quickwit-opentelemetry/src/otlp/traces.rs

```diff
@@ -291,7 +308,7 @@ impl Span {
 
 /// A wrapper around `Span` that implements `Ord` to allow insertion of spans into a `BTreeSet`.
 #[derive(Debug)]
-struct OrdSpan(Span);
+pub struct OrdSpan(Span);
 
 impl Ord for OrdSpan {
     fn cmp(&self, other: &Self) -> Ordering {
```

Comment:
nitpick...

Could you explain the motivation behind that order? 
Is it compression?